### PR TITLE
fix formatting and add bpmn Modeler connection

### PR DESCRIPTION
**Pull request by LaviniaStiliadou**
Signed-off-by: LaviniaStiliadou <st149989@stud.uni-stuttgart.de>
Use the following template for the PR title and delete this line: [WIP] Add BPMN Modeler

adds connection to the BPMN Modeler 

- Start and end date:
- Contributor:  Lavinia Stiliadou @LaviniaStiliadou, Timm Pankratz @Tpankv4 
- Supervisor: Kálmán Képes @nyuuyn 

- [x] Ensure that you followed our [toolchain guide](https://github.com/eclipse/winery/blob/master/docs/dev/github-workflow.md#github---prepare-final-pull-request).
- [x] Branch name checked. That means, the branch name starts with `thesis/`, `fix/`, ...
- [x] Ensure that the commit message is [a good commit message](https://github.com/joelparkerhenderson/git_commit_message)
- [x] Ensure to use auto format in **all** files
- [x] Ensure that you appear in `NOTICE` at Copyright Holders
- [ ] Tests created for changes
- [x] Documentation updated (see doc in [bowie](https://github.com/OpenTOSCA/bowie/tree/propUi/src/docs))
- [x] Screenshots added (see doc in [bowie](https://github.com/OpenTOSCA/bowie/tree/propUi/src/docs))

### CODE REVIEW

File: org.eclipse.winery.frontends/app/tosca-management/src/app/instance/admin/configuration/configuration.component.ts [hunk 1]
```diff
@@ -27,14 +28,59 @@ import { WineryNotificationService } from '../../../wineryNotificationModule/win
...
+    checkModelerAvailability() {
+        const containerURL = (<HTMLInputElement> document.getElementById('containerUrl2')).value;
+        if (containerURL) {
+            this.containerUrl = containerURL;
+            this.http.get(this.containerUrl, { responseType: 'text' }).subscribe(
+                () => this.containerUrlAvailable = true,
+                () => this.containerUrlAvailable = false
+            );
+        }
+        if (this.containerUrl) {
```
Comment (lharzenetter) [inline org.eclipse.winery.frontends/app/tosca-management/src/app/instance/admin/configuration/configuration.component.ts:60]: This shoud be `else if`, shoudn't it?

File: org.eclipse.winery.frontends/app/tosca-management/src/app/instance/admin/configuration/configuration.component.ts [hunk 2]
```diff
@@ -27,14 +28,59 @@ import { WineryNotificationService } from '../../../wineryNotificationModule/win
...
+                .subscribe(
+                    () => this.notify.success('Successfully saved Modeler URL!'),
+                    () => this.notify.error('Error while saving Modeler URL')
+                );
+        } else {
+            this.notify.error('Modeler URL is not available');
+        }
+    }
+
+    deleteModelerURL() {
```
Comment (lharzenetter) [inline org.eclipse.winery.frontends/app/tosca-management/src/app/instance/admin/configuration/configuration.component.ts:82]: Why would you delete the modeler URL?

File: org.eclipse.winery.frontends/app/tosca-management/src/app/instance/admin/configuration/configuration.component.ts [hunk 3]
```diff
@@ -27,14 +28,59 @@ import { WineryNotificationService } from '../../../wineryNotificationModule/win
...
+        if (this.containerUrl) {
+            this.http.get(this.containerUrl, { responseType: 'text' }).subscribe(
+                () => this.containerUrlAvailable = true,
+                () => this.containerUrlAvailable = false
+            );
+        }
+    }
+
+    saveModelerURL() {
+        if (this.containerUrlAvailable) {
```
Comment (lharzenetter) [inline org.eclipse.winery.frontends/app/tosca-management/src/app/instance/admin/configuration/configuration.component.ts:69]: Why don't you check the availability first? Maybe the check has not returend yet, even though it is availalbe, and thus an error is displayed.
But is okay for me now. Just keep it in mind.

**Review by nyuuyn — APPROVED** [review]
LGTM

**Review by lharzenetter — APPROVED** [review]

**Review by wederbn — APPROVED** [review]

**Comment by lharzenetter** [comment]
Now please create a PR at eclipse/winery an squash the commit. Also, please make sure, that your commit contains the sign-off and co-authored-by lines. Please see [the docu](https://github.com/OpenTOSCA/winery/blob/ustutt/docs/dev/github-workflow.md#github---prepare-final-pull-request) for details.

**Pull request closed without merging**